package.json

```diff
@@ -12,7 +12,8 @@
   "scripts": {
     "build": "fable-splitter -c splitter.config.js",
     "watch": "fable-splitter -c splitter.config.js -w",
-    "test": "fable-splitter -c splitter.test.config.js"
+    "test": "fable-splitter -c splitter.test.config.js",
+    "postinstall": "node ./node_modules/vscode/bin/install"

```

Comment:
This downloads the node_modules/vscode/vscode.d.ts based on the vscode engine version.